pessimistic-transaction.md

```diff
@@ -88,6 +82,8 @@ Pessimistic transactions in TiDB behave similarly to those in MySQL. See the min
 
 6. The data read by `EMBEDDED SELECT` in the statement is not locked.
 
+7. Open transactions do not block garbage collection (GC). By default, this limits the maximum execution time of pessimistic transactions to 10 minutes. You can modify this limit by editing `max-txn-ttl` under `[performance]` in the TiDB configuration file.
```

Comment:
A little bit confusing to me. Would you please help confirm?
- Does "Open transactions" mean the transactions that are being executed?
- Does "this" (in "this limits") mean all what is said in the previous sentence?
```suggestion
7. Open transactions in TiDB do not block garbage collection (GC). By default, this limits the maximum execution time of pessimistic transactions to 10 minutes. You can modify this limit by editing `max-txn-ttl` under `[performance]` in the TiDB configuration file.
```

Reply:
An "Open transaction" means a transaction that is in-flight (could have been started by `START TRANSACTION`, `begin` or `set autocommit=0` etc). Transactions have a read-view on the data, where they are not supposed to see changes. In MySQL this read-view is guaranteed to stay valid for basically forever. Garbage collection will pause to prevent cleaning it up. In TiDB the transaction will fail after 10 minutes.